data/LDS/projects.yaml

```diff
@@ -51,7 +54,7 @@ projects:
     type: Library
     license: Apache-2.0
     date_added: 2019-08-15
-    date_updated: 2021-10-21
+    date_updated: 2022-02-21
     maturity: 1
```

Comment:
I do not know how to populate this field: With the transfer of maintainer to Tune Insight, Lattigo is entering a new phase of development and will no longer be a research-only prototype.

Reply:
Our first external contributor! Woot!

The maturity is done using a spreadsheet like this: https://docs.google.com/spreadsheets/d/1c6lWKHK8wB1kRVJyqe_oh4BKPOiyypdGGWLqfGYA9tY/edit?usp=sharing

It has a link to the document describing how to assess the different boxes - do you want to have a go, or shall we re-do it? The last time we did a maturity evaluation for Lattigo was in 2019!

Reply:
I couldn't access the document linked from the spreadsheet. 

So I had a first shot and it seems we are already ticking a lot of boxes. I left my questions as comments but I will have a second pass if you grant me access to the descriptions.

Reply:
I replied and gave you access. TLDR: we have 0, 1, 2, or 3 as options.